akka-cluster/src/multi-jvm/scala/akka/cluster/MultiNodeClusterSpec.scala

```diff
@@ -26,34 +26,43 @@ import akka.actor.RootActorPath
 
 object MultiNodeClusterSpec {
 
+  /** run all tests using artery, rather than the "old" remoting */
+  val useArtery = sys.props.get("akka.cluster.test.use-artery").contains("true")
+  def arteryEnablerFragment = ConfigFactory.parseString("akka.remote.artery.enabled=on")
```

Comment:
We can point the jenkins job to only run tests in certain projects, e.g. akka-cluster and akka-cluster-tools. Then we should be able to run all with just artery.enabled=on